src/lib/libast/comp/omitted.c

```diff
@@ -1007,7 +1008,7 @@ ctime_now(const char* path)
 
 	if (sysstat(path, &fs) || (fs.st_mode & S_IWUSR) || syschmod(path, (fs.st_mode | S_IWUSR) & S_IPERM))
 		fs.st_mode = 0;
-	cygwin_conv_to_win32_path(path, tmp);
+	cygwin_conv_path(CCP_POSIX_TO_WIN_W | CCP_ABSOLUTE, path, tmp, 0);
```

Comment:
```suggestion
	cygwin_conv_path(CCP_POSIX_TO_WIN_A | CCP_ABSOLUTE, path, tmp, sizeof(tmp));
```
Similar for the other calls to cygwin_conv_path.

No idea where conv_path_list_buf_size (or  conv_last_list_buf_size) come from.  But if you need to convert a path list, you can use [cygwin_conv_path_list](https://cygwin.com/cygwin-api/func-cygwin-conv-path-list.html)

Reply:
so it looks like these old deprecated functions were 32-bit only, and removed in cygwin/cygwin@3e917daec121ae5fd5cdcbb25d0d4e9d1cfd0a2a